modules/functions.js

```diff
@@ -0,0 +1,260 @@
+// import { inventory } from "./inventory.js";
+
+// ---------------------- Referentie die gebruikt kunnen worden in de main.js. In dit document staan verder ook de uitwerkingen van de function van opdracht 3a t/m 4e.    -------------------------------------------------------------
+
+
+
+// //  --------------- Opdracht 3b - zie functions.js ----------
+
+
+// const tvBrandsInStore = dataInStore(inventory);
+// console.table(tvBrandsInStore);
+
+// const storeProperty = StoreDetails(inventory, 'brand');
+// console.table(storeProperty);
+
+
+
+// // ------------------ Opdracht 4a - zie functions.js ------------------
+  
+// const tvInfo = detailsTv(inventory, 'toshiba');
+// console.log(tvInfo);
+
+// // ------------------ Opdracht 4b - zie functions.js ------------------
+  
+// const tvCost = priceTv(inventory, 'toshiba');
+// console.log(tvCost);
+
+// // ------------------ Opdracht 4c - zie functions.js ------------------
+
+// const tvSize = screenSizeTv(inventory, 'toshiba');
+// console.log(tvSize);
+
+// // ------------------ Opdracht 4d - zie functions.js ------------------
+
+// const tvSpecs = TvSpecification(inventory, 'samsung');
+// console.log(tvSpecs);
+
+
+
+// -----------------------------------------------------------------------------------
+
+//  --------------- Opdracht 3a -------------
+
+// tv brands - overzicht lijst met tv-merken
+
+const tvBrands = inventory.map((inventoryBrand) => {
+
+  return inventoryBrand.brand;
+
+})
+
+console.table(tvBrands);
+
+//  function - array met data objecten
+
+function dataInStore (arr) {
+    
+  return arr;
+
+}
```

Comment:
Heeft deze functie nog een doel? Hier wordt namelijk teruggegeven wat in de functie binnenkomt

Reply:
Het gehele functions.js wordt eigenlijk niet gebruikt en was meer als een kladblok gebruikt als vertrekpunt voor de verschillende opdrachten. Delen hieruit in nieuwe .js files geplaatst.  Waarschijnlijk stond deze code er nog in. Heeft verder geen waarde alleen ter referentie.